utils/sampler/weighted_benchmark_test.go

```diff
@@ -119,14 +113,14 @@ func WeightedPowBenchmarkSampler(
 
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
-		_, _ = s.Sample(uint64(globalRNG.Int63n(int64(totalWeight))))
+		_, _ = s.Sample(globalRNG.Uint64n(totalWeight - 1))
 	}
 	return true
 }
 
 func WeightedSingletonBenchmarkSampler(b *testing.B, s Weighted, size int) bool {
 	weights := make([]uint64, size)
-	weights[0] = uint64(math.MaxInt64 - size + 1)
+	weights[0] = math.MaxUint64 - uint64(size+1)
```

Comment:
Why'd this go from `- size + 1` to `- size - 1`?

Reply:
:eagle: :eye: